pkg/cloudprovider/cloudapi/azure/azure_security_test.go

```diff
@@ -55,6 +55,7 @@ var _ = Describe("Azure Cloud Security", func() {
 		atAsgName                               = "atapplicationsgID"
 		agAsgID                                 = "nephe-ag-agapplicationsgID"
 		testPriority                      int32 = 100
```

Comment:
can you drop int32?

Reply:
Not really since Azure priority needs to be int32. I changed it to `testPriority = int32(100)` so it looks better.